tests/test_datasets/test_dataset_functions.py

```diff
@@ -340,3 +340,17 @@ def test_upload_dataset_with_url(self):
             url="https://www.openml.org/data/download/61/dataset_61_iris.arff")
         dataset.publish()
         self.assertIsInstance(dataset.dataset_id, int)
+
+    def test_data_status(self):
+        dataset = OpenMLDataset(
+            "UploadTestWithURL", "test", "ARFF",
+            version=1,
+            url="https://www.openml.org/data/download/61/dataset_61_iris.arff")
+        dataset.publish()
+        did = dataset.dataset_id
+
+        openml.datasets.status_update(did, 'active')
+        openml.datasets.status_update(did, 'deactivated')
+        openml.datasets.status_update(did, 'active')
```

Comment:
These calls only check if the function succeeds without crashing, but not whether it actually changes something on the server. Could you please check that something actually changed on the server.

Reply:
Added